docs/api.rst

```diff
@@ -2521,7 +2542,8 @@ of :class:`enum.Enum`.
         A member was kicked.
 
         When this is the action, the type of :attr:`~AuditLogEntry.target` is
-        the :class:`User` who got kicked.
+        the :class:`User` who got kicked. If the user is not found then it is
+        a :class:`Object` with an ID.
```

Comment:
```suggestion
        a :class:`Object` with the user's ID.
```
(same thing for the other ones)

Reply:
b34735ec07eb783a861f52c349f0153b6bc2e121